src/shared/seccomp-util.c

```diff
@@ -327,13 +337,27 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "lseek\0"
                 "pread64\0"
                 "preadv\0"
+                "preadv2\0"
```

Comment:
`preadv2` and `pwritev2` cannot be added when `2.3.1` is used. Would it make sense to bump the version 
 of `libseccomp` in `meson.build` to `2.3.2` so that `@basic-io` will try to filter the same set of syscalls everywhere?

Reply:
well, that'd be a mess with our CIs, because we'd first have to get a newer libseccomp version ready for them...

Note that libseccomp can be updated without the need to recompile systemd on this one. And if you do it will just start working, hence I don't think we need a hard requirement on that one...